typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -435,9 +436,13 @@ export abstract class HyperlaneAppGovernor<
   }
 
   async handleProxyAdminViolation(violation: ProxyAdminViolation) {
-    const code = await this.checker.multiProvider
-      .getProvider(violation.chain)
-      .getCode(violation.expectedProxyAdmin.address);
+    const provider = this.checker.multiProvider.getProvider(violation.chain);
+    const code = await provider.getCode(violation.expected);
+
+    const expectedProxyAdmin = ProxyAdmin__factory.connect(
```

Comment:
Instantiate the `expectedProxyAdmin` contract here instead of where violation is captured